eng/common/scripts/Helpers/ApiView-Helpers.ps1

```diff
@@ -169,6 +170,7 @@ function Set-ApiViewCommentForPR {
   $commentText += "## API Change Check"
   try {
     $response = Invoke-WebRequest -Uri $apiviewEndpoint -Method Get -MaximumRetryCount 3
+    LogDebug "OperationId: $($response.Headers['X-Operation-Id'])"
     if ($response.StatusCode -ne 200) {
       LogWarning "API changes are not detected in this pull request."
```

Comment:
Does this need to be a warning?

Reply:
I would only make this a warning if there is some action we expect for someone to take if they see this. If they have no action I would just keep it as a normal log so that we can use it if we need to debug why things aren't getting generated.